io/net/bonding.py.data/bonding.yaml

```diff
@@ -13,8 +13,11 @@ Test: !mux
         bonding_mode: "5"
     seventh:
         bonding_mode: "6"
-host_interfaces: "enP3p1s0f0,enP3p1s0f1"
-peer_ip: "85.85.85.85"
-peer_interfaces: "eth3"
+host_interfaces: "enP2p1s0f2,enP2p1s0f3"
```

Comment:
Please consider removing the interfaces and IP values from the yaml files when you send the next version for this PR